ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/HttpBinTest.kt

```diff
@@ -25,6 +25,9 @@ class HttpBinTest : ClientLoader() {
 
     @Test
     fun testGet() = clientTests {
+        // TODO: unmute when JS IR serialization is fixed https://github.com/Kotlin/kotlinx.serialization/issues/751
```

Comment:
How do we track this event? Who is responsible for this?

Reply:
> How do we track this event?

You can subscribe to linked issue

> Who is responsible for this?

TBD. @sandwwraith and Kotlin/JS team?